build.gradle

```diff
@@ -46,7 +47,7 @@ ext {
 
 allprojects {
     group = 'com.ververica.platform'
-    version = "2.3.2_${flinkVersion}"
+    version = "2.4.0_${flinkVersion}"
```

Comment:
Is this version referring to the corresponding VVP version? Then it should probably be 2.12.1.

If not, which logic we follow for the versioning?

Reply:
It is the version of the job/repo itself.  It connects to Flink versions, as you can see. 
I generally bump minor versions if we upgrade Flink versions, and bump patch versions in case of small/minor changes.